app/image_service.py

```
@@ -12,16 +11,57 @@
+        pass
+        
+    async def _execute_sql(self, query: str, params: tuple) -> bool:
         
-        self.db = db_instance
+        try:
+            async with db.get_async_connection() as conn:
+                cursor = conn.cursor()
+                cursor.execute(query, params)
+                conn.commit()
+                return True
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)